python/ray/_private/usage/usage_lib.py

```diff
@@ -269,6 +269,15 @@ def put_pre_init_usage_stats():
     _put_pre_init_extra_usage_tags()
 
 
+def reset_global_state():
+    global _recorded_library_usages, _recorded_extra_usage_tags
+
+    with _recorded_library_usages_lock:
+        _recorded_library_usages = set()
+    with _recorded_extra_usage_tags_lock:
+        _recorded_extra_usage_tags = dict()
```

Comment:
@rkooo567 @jjyao can I get a stamp for this small change pls? Only using it for testing.